pages/treeherder.py

```diff
@@ -120,66 +151,35 @@ def pin_using_spacebar(self):
         el.send_keys(Keys.SPACE)
         self.wait.until(lambda _: self.pinboard.is_pinboard_open)
 
+    def reset_filters(self):
+        """Filters Panel must be opened"""
+        self.find_element(*self._filter_panel_reset_locator).click()
+
     def select_mozilla_central_repo(self):
         # Fix me: https://github.com/mozilla/treeherder-tests/issues/43
         self.open_repos_menu()
         self.find_element(*self._mozilla_central_repo_locator).click()
         self.wait_for_page_to_load()
 
-    def select_random_repo(self):
-        self.open_repos_menu()
-        repo = random.choice(self.unchecked_repos)
-        repo_name = repo.text
-        repo.click()
-        self.wait.until(lambda s: self._active_watched_repo_locator == repo_name)
-        return repo_name
-
-    def click_on_filters_panel(self):
-        self.find_element(*self._filter_panel_locator).click()
-
-    def close_the_job_panel(self):
-        self.find_element(*self._close_the_job_panel_locator).click()
-
-    def deselect_all_failures(self):
-        """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_all_failures_locator).click()
-
-    def reset_filters(self):
-        """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_reset_locator).click()
-
-    def select_testfailed_failures(self):
-        """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_testfailed_failures_locator).click()
-
-    def deselect_testfailed_failures(self):
-        """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_testfailed_failures_locator).click()
-
     def select_busted_failures(self):
         """Filters Panel must be opened"""
         self.find_element(*self._filter_panel_busted_failures_locator).click()
 
-    def deselect_busted_failures(self):
-        """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_busted_failures_locator).click()
-
     def select_exception_failures(self):
         """Filters Panel must be opened"""
         self.find_element(*self._filter_panel_exception_failures_locator).click()
 
-    def deselect_exception_failures(self):
+    def select_testfailed_failures(self):
         """Filters Panel must be opened"""
-        self.find_element(*self._filter_panel_exception_failures_locator).click()
-
-    def checkbox_testfailed_is_selected(self):
-        return self.find_element(*self._filter_panel_testfailed_failures_locator).is_selected()
-
-    def checkbox_busted_is_selected(self):
-        return self.find_element(*self._filter_panel_busted_failures_locator).is_selected()
+        self.find_element(*self._filter_panel_testfailed_failures_locator).click()
 
-    def checkbox_exception_is_selected(self):
-        return self.find_element(*self._filter_panel_exception_failures_locator).is_selected()
+    def select_random_repo(self):
```

Comment:
It is not currently used, however it was intended to be in test_switch_repo.py. There was a problem with the data dependency, so we hard coded in which repo to select - but also kept this method for future use.